### PR TITLE
Setting up a sailfish phone jb60363 part2

### PR DESCRIPTION
**Pull request by jpwalden**

### CODE REVIEW

File: Support/Help_Articles/Setting_up_a_Sailfish_Phone_part2/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+title: Setting up a Sailfish phone part2
+permalink: Support/Help_Articles/Setting_up_a_Sailfish_phone_part2/
+parent: Help Articles
+layout: default
+nav_order: 805
+---
+
+This document is an continued tutorial for Sailfish OS setup, the first part of the setup is described in this [separate document](/Support/Help_Articles/Jolla_Store/Setting_up_a_Sailfish_phone/). Precondition is that the steps of the first tutorial are completed.
```
Comment (vigejolla) [inline Support/Help_Articles/Setting_up_a_Sailfish_Phone_part2/README.md:9]: There's an extra "Jolla_Store" in the link

Comment (jpwalden) [reply Support/Help_Articles/Setting_up_a_Sailfish_Phone_part2/README.md]: fixed

**Review by vigejolla — APPROVED** [review]
LGTM, but waiting for @jovirkku 's review before merging

File: Support/Help_Articles/Setting_up_a_Sailfish_Phone_part2/README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+---
+title: Setting up a Sailfish phone part 2
+permalink: Support/Help_Articles/Setting_up_a_Sailfish_phone_part2/
+parent: Help Articles
+layout: default
+nav_order: 805
+---
+
+This document is an continued tutorial for Sailfish OS setup, the first part of the setup is described in this [separate document](/Support/Help_Articles/Setting_up_a_Sailfish_phone/). Precondition is that the steps of the first tutorial are completed.
```
Comment (jovirkku) [inline Support/Help_Articles/Setting_up_a_Sailfish_Phone_part2/README.md:9]: ...is a continued tutorial...

(not a big problem, no need to fix if this is the only finding)

**Review by jovirkku — APPROVED** [review]

**Pull request merged**